cyberark/conjur-policy-generator
# K8s generator

### What does this PR do?

* adds a new generator based on the k8s demo
* updates tests, docs, rakefile, and UI to include the new demo
* updates capabilities of the policy template helpers to accommodate the new template

### Where should the reviewer start?

Most of the changes are in `generator.md`, where the policy generation helpers are enhanced and expanded, and the new template is defined.

Then observe there are also changes to the Rakefile and web UI to add the new template.

### How should the reviewer test this manually?

1. clone the repo and checkout this branch
2. run the tests: `bin/tests`
3. follow the instructions in `web/README.md` to set up a local dev environment and verify that the new generator works in the web UI
